osu.Game.Rulesets.Catch/Difficulty/CatchPerformanceCalculator.cs

```diff
@@ -90,6 +90,12 @@ public override double Calculate(Dictionary<string, double> categoryDifficulty =
             if (mods.Any(m => m is ModNoFail))
                 value *= 0.90;
 
+            if (categoryDifficulty != null)
+            {
+                categoryDifficulty.Add("AR", Attributes.ApproachRate);
+                categoryDifficulty.Add("Max Combo", Attributes.MaxCombo);
```

Comment:
When is this going to be used? Can this just be output via the difficulty attributes themselves when required (if it's intended for osu-tools)?

Reply:
These are indeed not required, removed